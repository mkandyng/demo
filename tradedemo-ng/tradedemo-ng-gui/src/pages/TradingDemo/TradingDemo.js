import React from "react";
import BottomLayout from "./bottomLayout/BottomLayout"
import TopLayout from "./topLayout/TopLayout"

export default function TradingDemo(props) {
  return (
    <div>
      <TopLayout />
      <BottomLayout />
    </div>
  )
};
