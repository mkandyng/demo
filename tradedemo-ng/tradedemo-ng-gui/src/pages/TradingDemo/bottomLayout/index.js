import BottomLayout from "./BottomLayout";
export default BottomLayout;
