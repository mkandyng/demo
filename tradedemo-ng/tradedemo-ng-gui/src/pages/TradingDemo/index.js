import TradingDemo from "./TradingDemo";
export default TradingDemo
