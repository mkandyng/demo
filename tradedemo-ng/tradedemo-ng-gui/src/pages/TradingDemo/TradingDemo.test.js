import React from "react";
import toJson from "enzyme-to-json";
import {shallow} from "enzyme";
import TradingDemo from "./TradingDemo";

/**
 * This is unit test of TradingDemo component
 * It leverage snapshot to test the presentation
 */

describe("TradingDemo", () => {

  it("should render component comparing with previous snapshot shallow markup", () => {
    // Given

    // When
    const component = shallow(<TradingDemo/>);

    // Then
    expect(toJson(component)).toMatchSnapshot();
  });

});
