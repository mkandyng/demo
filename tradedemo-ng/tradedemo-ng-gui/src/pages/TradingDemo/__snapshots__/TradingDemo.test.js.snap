// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TradingDemo should render component comparing with previous snapshot shallow markup 1`] = `
<div>
  <TopLayout />
  <Connect(BottomLayout) />
</div>
`;
